vote/src/vote_account.rs

```diff
@@ -58,6 +58,18 @@ pub struct VoteAccounts {
     >,
 }
 
+impl Clone for VoteAccounts {
+    fn clone(&self) -> Self {
+        Self {
+            vote_accounts: Arc::clone(&self.vote_accounts),
+            // Reset this so that if the previous bank did compute `staked_nodes`, the new bank
+            // won't copy-on-write and keep updating the map if the staked nodes on this bank are
+            // never accessed. See [`VoteAccounts::insert`] and [`VoteAccounts::staked_nodes`].
```

Comment:
nit: it's more often that `VoteAccounts::add_stake` and `VoteAccounts::sub_stake` update the staked nodes map than `VoteAccounts::insert`.

Reply:
Intriguing, I would have guessed that we process more votes than stake insns?

Reply:
Yeah but staked nodes is only modified if a vote account's node key is changed right? Not for normal votes

Reply:
err you're right!

Reply:
updated comment